recipes/qtpyinheritance/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "qtpyinheritance" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/klauer/qtpyinheritance/archive/v{{ version }}.tar.gz
+  sha256: 1a2194d75d49d2ec1839ab0db786c322ca67131109377f61419372b7ffd0f372
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - qtpy
+
+test:
+  imports:
+    - qtpyinheritance
+  requires:
+    - pyqt
+    - qt
```

Comment:
I suspect this isn't needed as you don't use it directly. It should be included as it is a dependencies of `pyqt`

Reply:
Addressed in 103967b